docs/learn.md

```diff
@@ -1,3 +1,153 @@
 # Learn
 
-_TODO: This will be a one-page with "all the things" in it_
+## New to OCaml?
+
+As Melange is a fork of the OCaml compiler, this documentation is designed to
+include the differences between both. But for the parts that overlap between
+Melange and OCaml, we will always refer to the main OCaml documentation
+throughout these docs.
+
+If you are completely new to OCaml, you might want to get familiar first with
+the language, there are [plenty of resources available](https://ocaml.org/docs),
+but we recommend the following tutorials from the official OCaml website:
+
+- [A First Hour With OCaml](https://ocaml.org/docs/first-hour)
+- [OCaml Exercises](https://ocaml.org/problems)
+
+## Why Melange
```

Comment:
I think this and the history paragraphs don't belong in the "Learn" page. IMO this page should be actionable and straight to the point.

We should make a separate page for "rationale" / "history".

Reply:
You suggested the other day to have all this content in a single page. The more content we add, the more I like this approach :) It allows to keep the navigation pretty minimal (Get started, Learn, API, Try, Community), so I am a bit unconvinced that history/rationale has enough weight to deserve its own top level section.

Would it be better if we move the Why & History to the bottom of the "Learn" page? That way, when users reach the page, they can go straight to the point as you say, but we aren't forced to add a new top-level section just for that.

Reply:
my suggestion was about navigation, i.e. make it possible to have a single page with all the text on the website, but that's not strictly necessary.

I'd still prefer if we keep logically separate docs in separate files. `mkdocs` doesn't allow to have a single page that concats all pages?

Reply:
> `mkdocs` doesn't allow to have a single page that concats all pages?

There is [some plugin](https://github.com/timvink/mkdocs-print-site-plugin) we could try.

> my suggestion was about navigation, i.e. make it possible to have a single page with all the text on the website, but that's not strictly necessary.
> I'd still prefer if we keep logically separate docs in separate files.

I think I misunderstood the whole thing, I can restructure the docs in separate files then. To clarify, the top sections would be:

- Get started
- Rationale <- new
- Learn
- API
- Try
- Community

?

Reply:
Moved in https://github.com/melange-re/documentation-site/commit/6cedebd48d2462bca74b4cee8d18952bd0d839d1.